python/GafferUI/MultiLineTextWidget.py

```diff
@@ -132,6 +133,14 @@ def getWrapMode( self ) :
 			QtGui.QTextOption.WrapAtWordBoundaryOrAnywhere : self.WrapMode.WordOrCharacter,
 		}[self._qtWidget().wordWrapMode()]
 
+	def setFixedLines( self, fixedLines ) :
```

Comment:
Turns out that is really hard to figure out these names, but I gotta say that I prefer your suggestion